test/tests/fuel-cycle/tests

```diff
@@ -7,4 +7,10 @@
     csvdiff = 'fuel_cycle_out.csv'
     requirement = 'The system shall reproduce a consistent solution to an ODE system of equations modeling the tritium fuel cycle.'
   []
+  [gui]
+    type = RunCommand
```

Comment:
@cticenhour please reach out for guidance on these more complex tests. `RunCommand` is very versatile and also very dangerous when it comes to usability in installed binaries.